### ISSUE TITLE
[Supporting the Platform - Make code - New Project (Editor)]: Focus indicator is not visible on Basic, Input, more, Music, Led...etc buttons in High contrast (Aquatic, Desert, Dusk, Night sky) themes

### ISSUE THREAD

**Issue by Chandangithb** [microsoft/pxt-microbit#5506]
### User Experience:
Color Sensitive people who rely on High contrast modes (to improve accessibility for vision impairments) will face difficulty in navigating the page if the focus indicator for any control is not adapting high contrast themes. As a result, they will be unable to identify the focused element.
**Note:**
User credentials should NOT be included in the bug.
### Prerequisite:
- Open System settings.
- Navigate to 'Contrast theme' using tab key and activate it.
- Navigate to 'Contrast theme' dropdown and select 'Aquatic, 'Desert', 'Dusk' and 'Night sky' mode.
- Navigate to 'Apply' button and activate it.

### Repro Steps:

1. Launch application URL : https://makecode.microbit.org/
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to search edit field.
6. Navigate to Basic, Input, more, Music, Led...etc buttons.
7. Verify whether focus indicator is visible on Basic, Input, more, Music, Led...etc buttons in high contrast theme or not.

### Actual Result:
**Issue:** Focus indicator is not visible on Basic, Input, more, Music, Led...etc buttons In High contrast (Aquatic, Desert, Dusk, Night sky) themes.
**Observation:** The issue is not repro with Application High contrast. Focus indicator is clearly visible on each control.

### Expected Result:
In High contrast (Aquatic, Desert, Dusk, Night sky) themes, focus indicator should be properly visible on Basic, Input, more, Music, Led...etc buttons. 

### MAS Reference:
[MAS 4.3.1 – No Disruption of Accessibility Features](https://aka.ms/MAS4.3.1)

### ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:
NA

### UIA/DOM details available:
NA

### Reference Links:

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Test Environment:

- OS Version: Windows 11 23H2 (OS Build:22631.3007)
- Browser: Chrome Version 121.0.2277.4 (Official build) dev (64-bit).
- Test Matrix: Chrome + JAWS
- URL: https://makecode.microbit.org/.

### Attachments:
**Application High Contrast:**
![MAS4 3 1-Application High Contrast](https://github.com/microsoft/pxt-microbit/assets/112375678/2febafe4-5785-4a21-9898-3d4a7aa0e40d)

**System High Contrast:**
![MAS4 3 1-Focus indicator is not visible on Basic, Input, more, Music, Led etc buttons in High contrast (Aquatic) themes](https://github.com/microsoft/pxt-microbit/assets/112375678/2e66934a-5f96-4d51-bb26-c153e520092b)
![MAS4 3 1-Focus indicator is not visible on Basic, Input, more, Music, Led etc buttons in High contrast (Desert) themes](https://github.com/microsoft/pxt-microbit/assets/112375678/1b0cc1d3-1b31-4296-bdac-7b39010e06f9)
![MAS4 3 1-Focus indicator is not visible on Basic, Input, more, Music, Led etc buttons in High contrast (Dusk) themes](https://github.com/microsoft/pxt-microbit/assets/112375678/1d5894ee-9d8c-473a-8b8d-c747bd12f3f9)
![MAS4 3 1-Focus indicator is not visible on Basic, Input, more, Music, Led etc buttons in High contrast (Night sky) themes](https://github.com/microsoft/pxt-microbit/assets/112375678/9325b43f-6512-4e4e-bcae-a876d22a4436)
![MAS4 3 1-Normal Mode](https://github.com/microsoft/pxt-microbit/assets/112375678/d6034636-f461-40ce-8df8-3a33c36f8a62)
